clean_all.py

```diff
@@ -341,6 +341,35 @@ def delete_all_instances(self):
                 )
 
 
+class BigTableClient(BaseDiscoveryClient):
+    endpoint = "bigtableadmin"
+    version = "v2"
+
+    def _delete_all_in_location(self, location: str):
+        raise NotImplementedError(
+            "BigTable is able to list all instances in one request."
+        )
+
+    def delete_all_instances(self):
+        logger.debug("Deleting BigTable instances in ALL locations")
+        instances = self._iterate(
+            endpoint=self.client.projects().instances(),
+            key="instances",
+            payload={"parent": f"projects/{self.project_id}"},
+        )
+
+        for instance in instances:
+            if SKIP_LABEL not in instance.get("labels", []):
```

Comment:
Please check if the instance is older than 24h, we try to keep this window

Reply:
AFAIK there is no such information available for `instance`.

https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/projects.instances#Instance